packages/types/src/package.ts

```diff
@@ -15,6 +24,7 @@ export interface Package {
       packages?: string[];
       platform?: "client" | "server";
       rendering?: "ssr" | "csr";
+      fills: { [key: string]: Fill };
```

Comment:
I'm not entirely sure if this is how I'm supposed to add the types here. @DAreRodz Do you have some experience with this?

Reply:
You put `fills` inside `state.frontity`. This is wrong, it's not `state.frontity.fills`, it is `state.fills`.

Reply:
Right, my bad, that's true. 

But assuming that I put them in `state.fills` my question was more about the general idea. Is this the right place to put the types for `fills`? I think so, but would just like to confirm.

Reply:
Yes, I think the placement and the types are right 👍